### PR TITLE
Nations-Facts First Release

### PR DESCRIPTION
**Pull request by iamsjunaid**
### In this milestone of the project, I've

- **Used API**:
I have used the [Rest Countries](https://restcountries.com/v3.1/all).

- **Pages**
It consists of the following pages:
1. **Home page**
It shows the list of countries when the page loads. It consists of a search bar that can filter the searched country from the countries list.

2. **Details page**
The details page shows the details of the selected country. For example the capital, area and the location of the country etc.

- **Testing requirements**
Wrote unit tests for pure functions in your Redux code.
Created integration tests for the application using the React Testing Library.

- **Technical requirements**
The project is a single-page application (SPA) built with React and Redux.
The data retrieved from the API should be stored in the Redux store.
You should filter the data that you retrieve from the API using a Filter stateless component.
Every page (the main page and the pages for each item) should have a unique route within the SPA.
The project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/).
Add a link to the online version of your app to your README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://nations-facts.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Walkthrough Video <a name="walkthrough-video"></a>
+
+- [Walkthrough Video Link](https://youtu.be/3sBy_mB1Bbo)
```
Comment (brytebee) [inline README.md]: Great video presentation which keeps to the required length. However, You should:
- highlight a piece of the code you find interesting or are proud of making.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @iamsjunaid 👋, 

##

### Highlights💡
- Good presentation and communication skills.✔️
- App is deployed and works properly.✔️
- All tests passed.✔️
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification. 
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by iamsjunaid** [comment]
I have updated the video link in the README.md file.

**Review by divinecharlotte — APPROVED** [review]
# Hello @iamsjunaid         😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive pr👍 
- Good implementation of the required changes👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by iamsjunaid** [comment]
Thank you for approving the PR😊.

**Pull request merged**